pom.xml

```diff
@@ -3,14 +3,13 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.16</version>
+    <version>4.55</version>
   </parent>
 
   <properties>
-    <jenkins.version>2.121.1</jenkins.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.361.4</jenkins.version>
+    <java.level>17</java.level>
     <findbugs.failOnError>false</findbugs.failOnError>

```

Comment:
The parent pom defines the `spotbugs.failOnError` property so that this is not needed.
```suggestion
```